data/risks/networks/1/strategylender.json

```diff
@@ -13,6 +13,8 @@
 			"genericlender"
 		],
 		"strategies": [],
-		"exclude": []
```

Comment:
i think this one needs to exclude the address 0x5F84acB4a27D762FC0E5CD6EFc1674827c0934C4 also

Reply:
added in b12932f